src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1266,10 +1251,17 @@ void LedgerConsensusImp::statusChange (
     JLOG (j_.trace()) << "send status change to peer";
 }
 
-void LedgerConsensusImp::takeInitialPosition (
-    std::shared_ptr<ReadView const> const& initialLedger)
+// For the consensus refactor, takeInitialPosition has been split
```

Comment:
The opening statement "for the consensus refactor" can be removed. Consider a comment that explains what the initial position is and why it's the not part of the consensus.